Diagrama de clases

celiabecerril opened the pull request.

celiabecerril committed:
Diagrama de clases

mmasias commented:
- ¿Capacidad como entidad?
- Hay dos sistemas, no uno (Jornada desconectada)
- ¿Por qué entrenador tiene flecha y jugadores no?
- ¿Dorsal como entidad?

mmasias committed:
imagenes diversas

mmasias committed:
Ejercicios

celiabecerril committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update ejercicio003.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

mmasias committed:
Update ejercicio003.md

mmasias committed:
Update ejercicio003.md

celiabecerril committed:
Merge branch 'mmasias:main' into main

celiabecerril committed:
Diagrama de clases Arcoiris

celiabecerril committed:
Diagrama de clases experiencia cliente

mmasias committed:
Añade reto de corrección para ejercicios

- Creado el archivo reto001.md con instrucciones para:
  - Corregir entregas anteriores según pautas de clase.
  - Documentar los ajustes de manera progresiva sobre los commits.
  - Enviar PR a la rama especificada una vez finalizado.

mmasias committed:
Añade tabla de retos con fechas en README

- Introducida una tabla con los detalles de los retos, incluyendo el enlace al reto 001, la fecha de entrega y la rama de entrega correspondiente.

celiabecerril committed:
Merge branch 'mmasias:main' into main

celiabecerril committed:
Mejora Experiencia Cliente 

Un cliente realiza una compra de un producto y  a su vez este producto es una venta de un vendedor que atiende al cliente 
La experiencia se refleja en una reseña que escribe un cliente sobre una compra y/o un vendedor

celiabecerril committed:
Mejoras Arcoiris

Una fuente de luz puede refractar agua o prisma y crear una descomposición de los colores (arcoiris) 
Creación de un observador que ve el arcoiris reflejado/proyectado en el cielo o en una superficie

mmasias committed:
Create infoClientes.md

celiabecerril committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update infoClientes.md

mmasias committed:
Update infoEquipos.md

celiabecerril committed:
Sistema de votacion

celiabecerril committed:
Delete entregas/SistemaVotacion directory

celiabecerril committed:
Add files via upload

celiabecerril committed:
Create Glosario.md

mmasias committed:
Update 1ra reunión

celiabecerril committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update fechas

mmasias committed:
Refactoring++

celiabecerril committed:
Add files via upload

celiabecerril committed:
Merge branch 'mmasias:main' into main

mmasias committed:
APP

mmasias committed:
APP

celiabecerril committed:
Merge branch 'mmasias:main' into main
